test/functional/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
```

Comment:
is there any way we can unify this with the regular test harness so we don't have to update script locations in more places?